.github/workflows/ci-cd.yaml

```
@@ -0,0 +1,163 @@
+      with:
+        filters: |
+          frontend:
+            - 'frontend-react/**'
+          backend:
+            - 'backend-flask/**'
+          vocab:
+            - 'vocab-importer/**'
+
+  quality-checks:
+    needs: changes
+    if: |
```

actionlint (1.7.4)

property "vocab-importer" is not defined in object type {backend: string; frontend: string; vocab: string}

(expression)